frontend/app/sysupport/detail/reload/template.hbs

```diff
@@ -0,0 +1,23 @@
+<h2>{{t 'sysupport.reload.select'}}</h2>
+<div uk-grid  uk-scrollspy="target: > div; cls:uk-animation-slide-bottom-small; delay: 50">
+  {{#each model as |package index|}}
+    <div class="uk-width-1-4@m uk-link-reset">
+      {{#uk-card hover=true as |card|}}
+        {{#card.header}}
+            {{#card.title}}{{format-duration package.duration}}{{/card.title}}
+        {{/card.header}}
+        {{#card.body}}
+          <dl class="uk-description-list">
+            <dt>{{t 'sysupport.reload.price'}}:</dt>
+            <dd>{{package.price}}</dd>
+          </dl>
+        {{/card.body}}
+        {{#card.footer}}
+          {{#uk-button data-test-project-reload=index class='uk-align-center' color='primary' on-click=(action send "subscribe" package)}}
```

Comment:
What is `send`?

Reply:
This calls [`send`](https://emberjs.com/api/ember/2.15/classes/Ember.Controller/methods/send?anchor=send) so i can define the action in the route.

Reply:
:+1: